CHANGES.txt

```diff
@@ -1,4 +1,59 @@
-﻿NUnit 3.0.0 Beta 2 - May 12, 2015
+﻿NUnit 3.0.0 Beta 3 - July ??, 2015
+
+Framework
+
+ * The RangeAttribute has been extended to support more data types including 
+   uint, long and ulong
+ * Added platform support for Windows 10 and fixed issues with Windows 8 and
+   8.1 support
+ * Added async support to the portable version of NUnit Framework
+ * Several TestCaseSourceAttribute constructors have been deprecated, see the
+   documentation for more information
+ * RandomAttribute has been extended to add support for new data types including 
+   uint, long, ulong, short, ushort, float, byte and sbyte
+ * Removed the dependency on Microsoft.Bcl.Async from the NUnit Framework assembly
+   targeting .NET 4.0. If you want to write async tests in .NET 4.0, you will need
+   to reference the NuGet package yourself.
```

Comment:
Possible addition: Significant improvements have been made in how NUnit deduces the type arguments of generic methods based on the arguments provided.

Does this feel like a "feature" to you? Do we want users to know that this has substantially changed?